data/core.yaml

```
@@ -1867,6 +1867,10 @@ en:
         message: "{feature} has no descriptive tags"
       relation_type:
         message: "{feature} is a relation without a type"
+    mutually_exclusive_tags:
+      title: Mutually Exclusive Tags
+      tip: "Find features having conflicting tags set."
+      message: "{feature} has conflicting tags: {tag1}, {tag2}. Only one should be set."
```

This warning message should be brief; provide more details in a `reference` property:

```suggestion
      title: Contradictory Tags
      tip: "Find features that have contradictory tags."
      message: "{feature} has both {tag1} and {tag2}"
      reference: "The keys {tag1} and {tag2} contradict each other. Only one should be set."
```

---

I updated the titles